sdk/lib/opentelemetry/sdk/version.rb

```diff
@@ -7,6 +7,6 @@
 module OpenTelemetry
   module SDK
     ## Current OpenTelemetry version
-    VERSION = '1.0.3'
+    VERSION = '2.0.0'
```

Comment:
```suggestion
    VERSION = '1.1.0'
```

Reply:
Now I am not so sure about this minor bump going out at the same time as the instrumentation changes.